Controller/MTGAController/Controller.py

```
@@ -1719,27 +1776,59 @@ def _get_last_scene_name(self) -> str | None:
+                    if m.start() < idx:
+                        owner = m.group(1)
+                    else:
+                        break
+                if owner is not None and owner != current_account:
+                    bot_logger.log_info(
+                        f"Quests block ignored: belongs to '{owner}' but current account "
+                        f"is '{current_account}' (stale after account switch)."
+                    )
+                    return None
+        except Exception:
+            pass
```

Ruff (0.15.20)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)